.vscode/settings.json

```diff
@@ -230,25 +19,53 @@
     {
       "name": "aila",
       "rootPath": "packages/aila"
+    },
+    {
+      "name": "api",
+      "rootPath": "packages/api"
+    },
+    {
+      "name": "core",
+      "rootPath": "packages/core"
+    },
+    {
+      "name": "db",
+      "rootPath": "packages/db"
+    },
+    {
+      "name": "eslint-config",
+      "rootPath": "packages/eslint-config"
+    },
+    {
+      "name": "prettier-config",
+      "rootPath": "packages/prettier-config"
+    },
+    {
+      "name": "exports",
+      "rootPath": "packages/exports"
+    },
+    {
+      "name": "ingest",
+      "rootPath": "packages/ingest"
+    },
+    {
+      "name": "logger",
+      "rootPath": "packages/logger"
+    },
+    {
+      "name": "rag",
+      "rootPath": "packages/rag"
     }
   ],
   "jest.runMode": "on-demand",
-  "jestrunner.jestCommand": "pnpm run test --",
-  "jestrunner.debugOptions": {
-    "runtimeExecutable": "${workspaceRoot}/node_modules/.bin/react-scripts",
-    "runtimeArgs": [
-      "test",
-      "${fileBasename}",
-      "--runInBand",
-      "--no-cache",
-      "--watchAll=false",
-      "--color"
-    ]
-  },
   "peacock.color": "8DA0D3",
-  "yaml.schemas": {
-    "file:///c%3A/Users/Simon/.vscode/extensions/atlassian.atlascode-2.8.6/resources/schemas/pipelines-schema.json": "bitbucket-pipelines.yml",
-    "https://www.artillery.io/schema.json": []
+  "typescript.preferences.preferTypeOnlyAutoImports": true,
+  "prettier.useEditorConfig": true,
```

Comment:
I can't see an editorconfig file in the project. Are you sure this is doing what you're expecting? 

https://prettier.io/docs/configuration#editorconfig